tests/QueuedViewExportTest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Maatwebsite\Excel\Tests;
+
+use Maatwebsite\Excel\Tests\Data\Stubs\Database\User;
+use Maatwebsite\Excel\Tests\Data\Stubs\AfterQueueExportJob;
+use Maatwebsite\Excel\Tests\Data\Stubs\SheetForUsersFromView;
+
+class QueuedViewExportTest extends TestCase
+{
+    /**
+     * Setup the test environment.
+     */
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->loadLaravelMigrations(['--database' => 'testing']);
+        $this->withFactories(__DIR__ . '/Data/Stubs/Database/Factories');
+    }
+
+    /**
+     * @test
+     */
+    public function can_queue_an_export()
```

Comment:
Can you add a test for multiple sheets as well. See the FromViewTest@can_export_multiple_sheets_from_view test as example.